.eslintrc

```diff
@@ -26,9 +28,12 @@
         "allowBlockStart": false
       }
     ],
+    "import/no-cycle": "off",
```

Comment:
Initially disabled rules!

* `import/no-cycle` shall be enabled in a follow-up PR as this one is already noisy enough as-is, and fixing it will require a lot of changes and there's potential for weirdness as I unpick the cycles.
* `import/no-named-as-default` I'm not really sure what to do with. This currently has lots of errors and fixing it may require rethinking how we we write imports/exports. It doesn't help that I think the rule itself might be a bit buggy.
* `shopify/jsx-no-complex-expressions` we probably want to keep disabled, This has 10 violations currently and in those cases we can't avoid it without some small rewrites that are more complex than I am comfortable putting in . PR with a bunch of brainless changes.

Reply:
All sounds good 👍 I think we might wanna create follow up issues for all of these so we don't forget about them.